sentry/src/main/java/io/sentry/SpanContext.java

```diff
@@ -119,6 +126,14 @@ public SpanContext(final @NotNull SpanContext spanContext) {
     if (copiedUnknown != null) {
       this.unknown = copiedUnknown;
     }
+    this.instrumenter = spanContext.instrumenter;
```

Comment:
just wanna check with @adinauer if this was just missed, or intended to not copy those?

Reply:
i talked with him and it was just missed, but it hasn't caused any issue.
Also, it will be removed in the future, so we can remove it from here